pandas/tests/reshape/merge/test_merge.py

```diff
@@ -2375,3 +2375,15 @@ def test_merge_right_left_index():
         }
     )
     tm.assert_frame_equal(result, expected)
+
+
+def test_merge_empty_right_index_left_on():
```

Comment:
can you do the exact reverse as well and parameterize over it (e.g. left_index=True)

Reply:
We are already testing both cases, parametrization is not really feasible here because we have to switch the dataframes too.
Renamed the test.